runtime/crab/src/pallets/bridge_grandpa.rs

```diff
@@ -16,14 +16,18 @@
 // You should have received a copy of the GNU General Public License
 // along with Darwinia. If not, see <https://www.gnu.org/licenses/>.
 
-pub use pallet_bridge_grandpa::Instance1 as WithDarwiniaGrandpa;
+pub use pallet_bridge_grandpa::Instance1 as WithPolkadotGrandpa;
 
 // darwinia
 use crate::*;
 
-impl pallet_bridge_grandpa::Config<WithDarwiniaGrandpa> for Runtime {
+frame_support::parameter_types! {
+	pub const PolkadotHeadersToKeep: u32 = 500;
```

Comment:
```
pub type PolkadotHeadersToKeep = ConstU32<500>;
```

Reply:
It needs to be used in two places, so I made it public and const.
https://github.com/darwinia-network/darwinia-2.0/blob/guantong%2Fbridge-pallets/runtime/crab/src/pallets/bridge_grandpa.rs#L30

https://github.com/darwinia-network/darwinia-2.0/blob/guantong%2Fbridge-pallets/runtime/crab/src/pallets/bridge_parachains.rs#L30

Reply:
`pub type PolkadotHeadersToKeep = ConstU32<500>;`

is also a public item.

Reply:
ok i just misunderstood